### DISCUSSION TITLE
Different publication dates in     D2.8.III.4 Data Specification on Land Use Technical Guidelines

### DISCUSSION THREAD

**Discussion by anamermar** [INSPIRE-MIF/helpdesk#201] [General]
Hello. Different publication dates in  :   D2.8.III.4 Data Specification on Land Use Technical Guidelines

 In the pdf document the publication date is 2024-07-31 but in the link the publication date is Publication date 31 January 2023

1/ Pdf -page 1:  https://knowledge-base.inspire.ec.europa.eu/publications/inspire-data-specification-land-use-technical-guidelines_en 

2/ Link:  https://inspire-mif.github.io/technical-guidelines/data/lu/dataspecification_lu.pdf

**Comment by fabiovinci** [comment]
Dear @anamermar,

thank you for notifying the error.
We will fix the typo as soon as possible.

**Comment by fabiovinci** [comment]
Dear @anamermar,

the errors have been corrected.
Thank you once again for bringing them to our attention.
